### PR TITLE
Add helpful message when receiving no first msg

### PR DESCRIPTION
**Pull request by janEbert**
If something goes wrong, this will give the user some idea of what is.

### CODE REVIEW

File: smach_viewer/scripts/smach_viewer.py [hunk 1]
```diff
@@ -645,7 +645,9 @@ def __init__(self):
         # Wait till we have received the first structure message with a top container
         with self._update_cond:
             while not self._top_containers:
-                self._update_cond.wait()
+                self._update_cond.wait(5.0)
```
Comment (rhaschke) [inline smach_viewer/scripts/smach_viewer.py]:
```suggestion
                if not self._update_cond.wait(5.0):
```
You should only print the message if the [timeout occured](https://docs.python.org/3/library/threading.html#threading.Condition.wait).

File: smach_viewer/scripts/smach_viewer.py [hunk 2]
```diff
@@ -645,7 +645,9 @@ def __init__(self):
...
+                print('Waiting for a structure message containing a '
```
Comment (rhaschke) [inline smach_viewer/scripts/smach_viewer.py]: Combine both lines, please. Setup your editor to accept a line width of 100 chars or so.
Given wide screens, the default limitation of 80 chars is too restrictive.

**Review by rhaschke — CHANGES_REQUESTED** [review]
Thanks for the PR. Some minor remarks remain.

**Comment by janEbert** [comment]
Resolved

**Review by rhaschke — APPROVED** [review]

**Pull request merged**